packages/babel-helper-define-polyfill-provider/src/visitors/usage.ts

```diff
@@ -24,7 +23,7 @@ export default (
       callProvider({ kind: "global", name }, path);
     },
 
-    MemberExpression(path: NodePath) {
+    MemberExpression(path: NodePath<t.MemberExpression>) {
       const key = resolveKey(path.get("property"), path.node.computed);
       if (!key || key === "prototype") return;
 
```

Comment:
(Line 31-32 below)

the type error is correct, we should only check the binding if `object.isIdentifier()`.

Reply:
added another conditional check for this